product_docs/docs/pgd/5/admin-tpa/upgrading_major_rolling.mdx

```diff
@@ -0,0 +1,537 @@
+Setting next multixact ID and offset for new cluster        ok
+Resetting WAL archives                                      ok
+Setting frozenxid and minmxid counters in new cluster       ok
+Restoring global objects in the new cluster                 ok
+Restoring database schemas in the new cluster               ok
+Copying user relation files                                 ok
+Setting next OID for new cluster                            ok
+Sync data directory to disk                                 ok
+Creating script to delete old cluster                       ok
+Checking for extension updates                              notice
+
+Your installation contains extensions that should be updated
+with the ALTER EXTENSION command.  The file
+    update_extensions.sql
+when executed by psql by the database superuser will update
+these extensions.
+
+
+Upgrade Complete
+----------------
+Optimizer statistics are not transferred by pg_upgrade.
+Once you start the new server, consider running:
+    /usr/pgsql-15/bin/vacuumdb --all --analyze-in-stages
+
+Running this script will delete the old cluster's data files:
+    ./delete_old_cluster.sh
+
+pg_upgrade complete, performing BDR post-upgrade steps
+------------------------------------------------------
+Collecting old cluster control data                         ok
+Collecting new cluster control data                         ok
+Checking LSN of new cluster                                 ok
+Starting new cluster with BDR disabled                      ok
+Connecting to new cluster                                   ok
+Creating replication origin (bdr_bdrdb_rb69_bdr2)           ok
+Advancing replication origin (bdr_bdrdb_rb69_bdr2, 0/1F4... ok
+Creating replication origin (bdr_bdrdb_rb69_bdr1)           ok
+Advancing replication origin (bdr_bdrdb_rb69_bdr1, 0/1E8... ok
+Creating replication slot (bdr_bdrdb_rb69_bdr1)             ok
+Creating replication slot (bdr_bdrdb_rb69)                  ok
+Creating replication slot (bdr_bdrdb_rb69_bdr2)             ok
+Stopping new cluster
+```
+
+### Modify the Postgres service
+
+The Postgres service on the system is configured to start the old version 
+of Postgres (PG15). You need to modify the `postgres.service` file to start the new 
+version (PG16). You can do this using `sed` to replace the old version
+number `15` with `16` throughout the file. 
+
+After you've changed the version number, you can tell the 
+systemd daemon to reload the configuration. On the target node, run:
+
+```bash
+sudo sed -i -e  's/15/16/g' /etc/systemd/system/postgres.service
+sudo systemctl daemon-reload
+```
+
+### Start Postgres
+
+Start the modified Postgres service by running:
+
+```bash
+sudo systemctl start postgres
+```
+
+### Validate the Postgres version
+
+Repeating the first step, check the version of Postgres to confirm that you upgraded kaboom correctly. While still on kaboom, run:
+
+```bash
+sudo -u postgres psql bdrdb
+```
+
```

Comment:
Or just `select version()` to return that nodes version? Might be simpler and less info to parse.